docusaurus.config.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
             },
             {
               to: "/application-development",
-              activeBasePath: "/application-development",
+              activeBaseRegex: "(/application-development)|(/docs/(go|java|php|node))",
```

Comment:
@sw-yx this permits to have the "SDKs" dropdown item to be highlighted when browsing any of the 4 SDKs. Docusaurus can't guess about those SDKs by itself, it just sees a regular page.

![image](https://user-images.githubusercontent.com/749374/125669431-8d721021-cecd-4e78-9fa7-e05eb8c6a484.png)


New mobile collapsible auto-expansion implemented in https://github.com/facebook/docusaurus/pull/5168:

![image](https://user-images.githubusercontent.com/749374/125669255-e6e36ed6-7ec4-4ef2-8567-9e5a88336447.png)

(seen after navigating back to main sidebar menu)

Reply:
thats so smart! I never thought to use it like that.